### DISCUSSION TITLE
Plantage de l'interface quand on choisit "principales compétitions" dans Football

### DISCUSSION THREAD

**Discussion by kreon76** [pretrehr/Sports-betting#24] [General]
Bonjour, pour info : Plantage de l'interface quand on choisit "principales compétitions" dans Football

![image](https://user-images.githubusercontent.com/78261069/108312876-9eaca300-71b7-11eb-989d-7d5c2c0cd544.png)

Merci par avance

Cdt

**Comment by pretrehr** [comment]
Exact, merci !
Je règlerai le problème dans la journée. En attendant, vous pouvez cliquer d'abord sur "Compétitions actuelles", puis "Oui" sur la fenêtre affichée, puis "Principales compétitions". (Il suffit de le faire une fois, ensuite vous pourrez utiliser à nouveau directement "Principales compétitions".)

**Reply by kreon76** [reply]
Nickel ça fonctionne parfaitement!
